### PR TITLE
Add RepaymentHelper contract and update cli to call the function on Alfajores

### PR DESCRIPTION
**Pull request by x-moola**
### Description

For [AMM-1332 [Solidity] Create a helper repayment contract ](https://moolamarket.atlassian.net/browse/AMM-1332).

Add RepaymentHelper contract that:
repays debt on behalf of another account, if the specified amount is more than the actual debt, deposits the remaining amount to the delegatee. 

------ 
Deployed one on Alfajores:
[0x954234d95900AD58fAB116EcF6a454b4C3255913](https://alfajores-blockscout.celo-testnet.org/address/0x954234d95900AD58fAB116EcF6a454b4C3255913/transactions)

### CODE REVIEW

**Comment by lastperson** [comment]
Contract is good, just need to add tests and deploy.

**Comment by x-moola** [comment]
Tests will be added in a separate PR

File: cli.js [hunk 1]
```diff
@@ -462,37 +474,26 @@ async function execute(network, action, ...params) {
     }
     console.log(
       'Approve',
-      (
-        await token.methods
-          .approve(lendingPool.options.address, amount)
-          .send({ from: user, gas: 2000000 })
-      ).transactionHash
+      (await token.methods.approve(lendingPool.options.address, amount).send({ from: user, d }))
```
Comment (lastperson) [inline cli.js:477]: something seem to be broken with 'd' ?

File: cli.js [hunk 2]
```diff
@@ -1558,6 +1544,38 @@ async function execute(network, action, ...params) {
...
+    const assetAddr = asset.options.address;
+    const amount = web3.utils.toWei(params[2]);
+    const rateModeInput = params[3];
+    const user = params[4];
+
+    if (!isValidRateMode(rateModeInput)) return;
+    const rateMode = getRateModeNumber(rateModeInput);
+
+    if (privateKeyRequired) {
+      pk = process.env.CELO_BOT_PK || params[5];
```
Comment (lastperson) [inline cli.js:1559]:
```suggestion
      pk = params[5];
```

File: cli.js [hunk 3]
```diff
@@ -1558,6 +1544,38 @@ async function execute(network, action, ...params) {
...
+      if (!pk) {
+        console.error('Missing private key');
+        return;
+      }
+      kit.addAccount(pk);
+    }
+
+    try {
+      const repayDelegationCallTx = await repayDelegationHelper.methods
```
Comment (lastperson) [inline cli.js:1568]: should check/give approvals before calling repay.

**Review by lastperson — CHANGES_REQUESTED** [review]

File: cli.js [hunk 4]
```diff
@@ -1549,10 +1568,68 @@ async function execute(network, action, ...params) {
...
+      kit.addAccount(pk);
+    }
+
+    const repayDelegationHelperAddress = repayDelegationHelper.options.address;
+    try {
+      // check and approve spend of the tokens
+      const currentAllowance = await asset.methods
+        .allowance(user, repayDelegationHelperAddress)
+        .call();
+      if (currentAllowance < amount) {
```
Comment (lastperson) [inline cli.js:1606]: those value are either string or big numbers, in both cases lessThan operator wouldn't work as expected.

**Pull request merged**